docker/Dockerfile

```diff
@@ -29,10 +29,25 @@ RUN set -e; \
     esac; \
     truncate -s0 ~/.bash_history;
 
+# - On Ubuntu 24.04 dstat/pcp cannot be installed noninteractively after
+#   installed software-properties-common.
+#   Install it first as a workaround.
+#   https://bugs.launchpad.net/ubuntu/+source/pcp/+bug/2064784
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
     . <(sed 's/^\(..*\)/export DISTRO_\1/' '/etc/os-release'); \
     case "$DISTRO_ID-$DISTRO_VERSION_ID" in \
+    'ubuntu-24.04') \
+        for attempt in $(seq "$DEB_MAX_ATTEMPT" -1 0); do \
+            [ "$attempt" -gt 0 ]; \
+            sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+                dstat \
```

Comment:
I saw dstat pkg has been removed from 24.04 and it's part of pcp.

Reply:
Please simply switch to pcp instead, for all distro versions.
All ubuntu since 20.04 provides pcp pkg, with dstat cmd included, and marked as conflicting with dstat pkg.